packages/core/src/components/pill/pill.tsx

```diff
@@ -64,6 +65,13 @@ export class Pill implements IxComponent {
    */
   @Prop() alignLeft = false;
 
+  /**
+   * Optional text to show as a tooltip (title attribute).
+   * By default the component text content will be used. Set to 'none' to not show a tooltip.
+   * @since 3.0.0
+   */
+  @Prop() tooltipText?: string;
```

Comment:
`string` type is confusing here. It should general be `string | 'none'`. Imo better solution would be to explizit set the variable itself to `false` or `''`

```
<ix-chip> // content is tooltip
<ix-chip tooltip-text={false}> // no tooltip
<ix-chip tooltip-text={''}> // no tooltip
<ix-chip tooltip-text={'xyz'}> // 'xyz' tooltip
```

Reply:
Valid points. The ACs explicitly state 'none' as option. We may have missed that during refinement.